### PR TITLE
Validate individual packages

### PR DESCRIPTION
**Pull request by alexkilgour**
Add command line argument to allow validation of a single package instead of always running on all the packages

```
// validate all packages
$ npm run validate

// validate only fec-package
$ npm run validate fec-package
```

### CODE REVIEW

File: scripts/validate.js [hunk 1]
```diff
@@ -14,29 +14,48 @@ const getLicense = require('./_get-license');
...
-							});
-					}).catch(err => {
-						exitScript.displayErr(err);
-					});
-			}).catch(err => {
-				exitScript.displayErr(err);
-			});
-	}).then(() => i >= lastPackagePathIndex || pathLoop(i + 1));
-})(0);
+if (filteredPaths.length > 0) {
```
Comment (jpw) [inline scripts/validate.js]: could we reduce this flying V of JS please? like bailing early on this if condition?

File: scripts/validate.js [hunk 2]
```diff
@@ -14,29 +14,48 @@ const getLicense = require('./_get-license');
...
+	(function pathLoop(i) {
```
Comment (jpw) [inline scripts/validate.js]: do we need IIFE's in node?

File: scripts/validate.js [hunk 3]
```diff
@@ -14,29 +14,48 @@ const getLicense = require('./_get-license');
...
 const exitScript = require('./_exit-script');
 
 const allPackagePaths = getPackages(path.resolve(__dirname, '../packages'));
-const lastPackagePathIndex = allPackagePaths.length - 1;
 const globalLicense = getLicense('../package.json');
+const lastArgument = process.argv[process.argv.length - 1];
+const filteredPaths = filterPaths(lastArgument);
+const lastPackagePathIndex = filteredPaths.length - 1;
+
+// Filter the package list based on CLI argument
```
Comment (jpw) [inline scripts/validate.js]: improve function name, rm comment?

**Comment by alexkilgour** [comment]
col, thanks. will take a look at those

File: scripts/validate.js (hunk 2 above)
Comment (alexkilgour) [inline scripts/validate.js]: It's not needed in node, but i'm using the IIFE syntax to pass the `0` starting value

Comment (jpw) [inline scripts/validate.js]: is the version of node you are targetting compatible with ES6 default parameters?

Comment (alexkilgour) [inline scripts/validate.js]: yes, but you've still got to call the function right? Anyway, i'm having to rewrite it to get over the other issue so this is going anyway 😆

File: scripts/validate.js (hunk 3 above)
Comment (alexkilgour) [inline scripts/validate.js]: 👍

**Pull request merged**